scripts/airflow3/setup.sh

```diff
@@ -4,26 +4,17 @@
 set -e
 
 # Create a UV virtual environment named 'env' (you can change this as needed)
-echo "Creating UV virtual environment at $(pwd)/tools"
-uv venv venv --directory "$(pwd)/scripts/airflow3"
+echo "Creating virtual environment at $(pwd)/tools"
+python3 -m venv "$(pwd)/scripts/airflow3/venv"
```

Comment:
We'll probably need to make this change to more places, but it would make our local development experience easier, so we can easily spot if we're on AF2 or AF3:
```suggestion
python3 -m venv "$(pwd)/scripts/airflow3/venv-af3"
```